files/forms.py

```
@@ -169,17 +203,60 @@ def clean_uploaded_poster(self):
+                    # Use the custom year as the final value
+                    cleaned_data['year_produced'] = year_custom
+
+        elif year_produced:
+            # User selected from the dropdown, validate that it is a valid year
+            try:
+                year_int = int(year_produced)
+                current_year = datetime.now().year
+                # Assuming the dropdown years start from 2000
+                if not (2000 <= year_int <= current_year):
+                    self.add_error('year_produced',
+                                  f'Please select a valid year.')
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)